### ISSUE TITLE
join planetary.name room broken due to bypass-invites-token not being set on server / in app

### ISSUE THREAD

**Issue by rabble** [planetary-social/planetary-ios#947]

**Comment by boreq** [comment]
Related to #886?

**Comment by mplorentz** [comment]
> Related to #886?

Yes for sure. Two different issues but both need to be fixed before we can programmatically join Planetary.name.

**Comment by mplorentz** [comment]
I confirmed today that when the app is properly deployed with `planetaryRoomToken` set in its Secrets file then it matches the token deployed on planetary.name and the whole flow works correctly. I'm not sure if the latest production deployment of Planetary contains the `planetaryRoomToken` so I'm not closing this issue yet.
